Parser/parser.py

```diff
@@ -195,13 +210,25 @@ def Compile(code:list) -> str:
 
             case NodeTypes.VAR_DECL:
                 node = parse_VarDecl(line)
-                code_string += node.visit() + ";\n"
+                code_string += node.visit() + "\n"
```

Comment:
why you not add "\n" in visit() method is there is  any reason for that

Reply:
yes because adding a line during visiting will resist the use of visitor in other areas to avoid such this i decided to add during the line by line conversion only.